src/entities/profile/ui/ProfileOverview.tsx

```diff
@@ -0,0 +1,105 @@
+import { useState } from "react";
+import { EditIcon, DropDownIcon } from "@/shared/ui/icon";
+
+// TODO alt 추가하기
+export const ProfileImage = ({ src }: { src: string }) => {
+  return <img src={src} alt="profile" className="w-16 h-16 rounded-[28px]" />;
+};
+
+interface ProfileInfoProps {
+  profileInfo: {
+    name: string;
+    breed: string;
+    follower: number;
+    following: number;
+  };
+}
```

Comment:
profileInfo key를 넣은 이유가 있을까요?!
아래처럼 할 수도 있을 것 같아서요!

```suggestion
interface ProfileInfoProps {
    name: string;
    breed: string;
    follower: number;
    following: number;
}
```

Reply:
객체 형태로 넣은 이유는 컴포넌트 `props` 에 값을 넣을 때 

하나 하나 나열해서 넣는 편보다 객체 하나로 해서 하나의 프로퍼티로 넣는 편이 낫다고 생각했어요 

```tsx
// 하나씩 꺼내 넣는 경우
<ProfileInfo name = {} breed = {} follower = {} following = {}>

<ProfileInfo profileInfo = {}>
```

그런데 생각해보니 그냥 스프레드 연산으로 넣는 방법이 있겠군요 

반영해서 수정하겠습니다.